Cartfile

```diff
@@ -0,0 +1 @@
+github "ilammy/OpenSSL" ~> 1.0.2
```

Comment:
if we need some fork of openssl, maybe fork it to cossacklabs/* and use from same organization repositories?

Reply:
we hope that Marcin will merge our PR

https://github.com/krzyzanowskim/OpenSSL/pull/55

if this will happen soon – this dependency will be re-linked to the original repo.

otherwise, we will fork openssl into `cossacklabs/*`

Reply:
Confirming that's the plan. We'll do a proper fork before releasing 0.11 if we are not able to use upstream by that time.